src/components/commandPallete.js

```diff
@@ -1,7 +1,7 @@
 import tag from 'html-tag-js';
 import Commands from '../lib/ace/commands';
 import helpers from '../lib/utils/helpers';
-import inputhints from './inputHints';
+import inputhints from './inputhints';
 
```

Comment:
No change needed

Reply:
This file also requires change, without it webpack produces this error:
```
ERROR in ./src/components/commandPallete.js 6:0-38
Module not found: Error: Can't resolve './inputHints' in '/home/Acode/src/components'
resolve './inputHints' in '/home/Acode/src/components'
  using description file: /home/Acode/package.json (relative path: ./src/components)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/Acode/package.json (relative path: ./src/components/inputHints)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/Acode/src/components/inputHints doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/Acode/src/components/inputHints.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /home/Acode/src/components/inputHints.json doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /home/Acode/src/components/inputHints.wasm doesn't exist
      as directory
        /home/Acode/src/components/inputHints doesn't exist
 @ ./src/lib/commands.js 33:0-81 69:4-19
 @ ./src/lib/acode.js 11:0-34 73:17-25 74:15-23
 @ ./src/lib/main.js 56:0-28 193:31-36

webpack 5.73.0 compiled with 1 error in 57474 ms
```